src/compressed_tensors/quantization/observers/helpers.py

```diff
@@ -13,18 +13,55 @@
 # limitations under the License.
 
 from collections import Counter
-from typing import Tuple
+from typing import Optional, Tuple
 
 import torch
 from compressed_tensors.quantization.quant_args import (
     FP8_DTYPE,
     QuantizationArgs,
+    QuantizationStrategy,
     QuantizationType,
 )
 from torch import FloatTensor, IntTensor, Tensor
 
 
-__all__ = ["calculate_qparams", "get_observer_token_count", "calculate_range"]
+__all__ = [
+    "calculate_qparams",
+    "get_observer_token_count",
+    "calculate_range",
+    "compute_memoryless_zp_and_scales",
+]
+
+
+def compute_memoryless_zp_and_scales(value: Tensor, args: QuantizationArgs):
```

Comment:
Can this function be reused, and if so should it be renamed and support more strategies?

Reply:
Should only be used for dynamic case, which isn't supported beyond per token

Reply:
Maybe rename to `calculate_dynamic_qparams` to make it clear how this function should be used